LINK/usr/bin/vmdb-up

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env ruby
+
+require "yaml"
+require "sd_notify"
+require "manageiq-password"
+
+$stdout.sync = true
+
+def wait_for_service(host, port)
+  loop do
+    `ncat #{host} #{port} </dev/null 2>/dev/null`
+    break if $?.success?
+
+    puts "#{host} #{port} - not accepting connections"
+    sleep(10)
+  end
+
+  puts "#{host} #{port} - accepting connections"
+end
+
+def wait_for_database(db_yaml, env = "production")
+  host, port = db_yaml[env].values_at("host", "port")
+  wait_for_service(host, port || 5432)
+end
+
+def wait_for_vmdb(db_yaml, env = "production")
+  host, port, dbname, password = db_yaml[env].values_at("host", "port", "database", "password")
+  password = ManageIQ::Password.decrypt(password)
+
+  loop do
+    results = `PGPASSWORD=#{password} psql --host=#{host} --port=#{port || 5432} --dbname=#{dbname} --command="SELECT FROM miq_regions LIMIT 1"`
+    break if $?.success? && results.include?("(1 row)")
+
+    puts "VMDB not ready yet"
+    sleep(10)
+  end
+
+  puts "VMDB is up and running"
+end
+
+database_yml = YAML.load_file("/var/www/miq/vmdb/config/database.yml")
```

Comment:
do we want to allow updating of this?

I'm guessing that on an upgrade the db won't change.

I'm guessing on a new server this will change. BUT, we won't try and start this service until after it is set, so this should be good